### ISSUE TITLE
Add Blamable Behavior and Timestamp Behavior

### ISSUE THREAD

**Issue by radiegtya** [noumo/easyii#29]
I think easyii need blamable behavior and timestamp behavior for each news, article etc.

That's very important I think.

Thanks

**Comment by noumo** [comment]
There is no users in easyii, so i dont think blamable behavior is needed.
Timestamp behavior, yes this is good idea. Thank you.

**Comment by radiegtya** [comment]
yes, easyii have users which is admin. please take a look at my code here. (i have adding api and model in News module). also I had adding migration. 

https://github.com/radiegtya/easyii/tree/master/modules/news

if you are interested, I'll send You PR.

**Comment by noumo** [comment]
Im interested but a little later, because in 1-2 weeks will be big update with a lot of changes in API and structure, so please lets postpone this feature.
